pkg/images/builder.go

```diff
@@ -76,8 +76,8 @@ func Build(ctx context.Context, client docker.ImageAPIClient, dir, name string,
 		return errors.Wrap(err, "failed to create a tarball archive")
 	}
 
-	log.Debugf("Building image %s from tarball", name)
-	r, err := client.ImageBuild(ctx, tarBall, types.ImageBuildOptions{
+	log.Infof("Building image %s from tarball", name)
+	r, err := client.ImageBuild(context.Background(), tarBall, types.ImageBuildOptions{
```

Comment:
Was this intentional? why is this needed?

Reply:
not intentional at all